doc/samtools-view.1

```diff
@@ -349,6 +349,9 @@ of input.
 Include customized index file as a part of arugments. See
 .B EXAMPLES
 section for sample of useage.
+.TP
+.BI --no-PG
```

Comment:
`.BI` is for when there's an option and an argument: `.BI --boldme italicme`. There's only bold here, so `.B --no-PG`.